tools/mysql/schema-update-6.sql

```diff
@@ -0,0 +1,10 @@
+-- !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+-- ALWAYS BACKUP YOUR DATA BEFORE EXECUTING THIS SCRIPT
+-- !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+# This script upgrade DB schema from version 5 to version 6
+
+ALTER TABLE builds ADD COLUMN ensure_trusted_build TINYINT(1) NOT NULL DEFAULT 0;
```

Comment:
your sql file "deploy.sql" change is in environs table

Reply:
Updated.